### ISSUE TITLE
Tool idea: Paketti slicer step seq

### ISSUE THREAD

**Issue by dmt-hub** [esaruoho/paketti#318]
Set the slice value (0Sxx) or note value (tied to instrument) in this box 
![afbeelding](https://github.com/user-attachments/assets/292979c5-19e1-438b-a539-3d20672e1496)

step sequencer for that slice/note:
![afbeelding](https://github.com/user-attachments/assets/71a4d395-8b17-4bbe-a75c-ebe7f52fbd3c)

ability to set loop point for each slide seq - to do polyrhythms (see gater)
Shift notes (see gater)
print

add these options for 8 or more slices in 1 tool window.

**Comment by esaruoho** [comment]
so this would be 16 rows with one valuebox on every row, right?

**Comment by dmt-hub** [comment]
yes one valuebox per row - where one can choose slicenumber or note
also valuebox to set the looppoint for each row
16 rows is idd maybe better if you have a breakbeat with more then 8 slices

**Comment by esaruoho** [comment]
@dmt-hub hi.. so i was wondering.. wanna tell me what i should be doing if i have 32 slices, 64 slices, 128 slices or 256 slices?
just create more rows?
